### PR TITLE
Add blog post introducing points

### PR DESCRIPTION
**Pull request by FionaDL**
**IMPORTANT: blog posts must be approved by two reviewers before they can be published, one of which must be a native English speaker.**

Here is a blog post introducing points, I am adding a WIP label until @ABizzinotto  has a change to review and add anything that she may want to include.

### CODE REVIEW

**Comment by ABizzinotto** [comment]
@FionaDL I added my updates here :) It's ready for you to review and, if everything looks good, remove the WIP label.

**Comment by FionaDL** [comment]
@lubc this is ready to be reviewed.

**Review by ABizzinotto — APPROVED** [review]

File: _posts/2020-10-01-introducing-points.markdown [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+## After the Project
+Let’s say all goes well, and the client approves the proposal and the project moves forward! This will be a chance to gather real data on how long and how complex stories really were. This is why we have also created a section in the admin panel for entering actual points.
+
+<img src="/blog/assets/images/points/real-score-page.png" alt="real score points ombulabs" class="medium-img">
+
+Later we can use the actual point data to become better estimators on future projects. You can [find more information on that process here](https://www.ombulabs.com/blog/agile/scrum/become-better-estimators.html).
+
+## Conclusion
+Points in a blind estimation app allowing our team to create project proposals that are more spot on. Points is open source and an ongoing project. We welcome contributions, and hope that our project can be useful to you!
```
Comment (lubc) [inline _posts/2020-10-01-introducing-points.markdown:48]: "Points in a blind estimation app allowing our team to create project proposals that are more spot on."

@FionaDL can you review this sentence? something sounds off to me

Comment (FionaDL) [reply _posts/2020-10-01-introducing-points.markdown:48]:
```suggestion
Points in a blind estimation app that allows our team to create project proposals that are more spot on. Points is open source and an ongoing project. We welcome contributions, and hope that our project can be useful to you!
```

File: _posts/2020-10-01-introducing-points.markdown [hunk 2]
```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: "Introducing Points - An blind estimation tool from OmbuLabs"
+date: 2020-10-16 12:00:00
+categories: ["open-source", "agile"]
+author: "fionadl"
```
Comment (FionaDL) [inline _posts/2020-10-01-introducing-points.markdown:6]:
```suggestion
author: ["fionadl", "abizzinotto"]
```

Comment (lubc) [reply _posts/2020-10-01-introducing-points.markdown:6]: @FionaDL should be `authors` (plural) to work

File: _posts/2020-10-01-introducing-points.markdown [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+## After the Project
+Let’s say all goes well, and the client approves the proposal and the project moves forward! This will be a chance to gather real data on how long and how complex stories really were. This is why we have also created a section in the admin panel for entering actual points.
+
+<img src="/blog/assets/images/points/real-score-page.png" alt="real score points ombulabs" class="medium-img">
+
+Later we can use the actual point data to become better estimators on future projects. You can [find more information on that process here](https://www.ombulabs.com/blog/agile/scrum/become-better-estimators.html).
+
+## Conclusion
+Points in a blind estimation app that allows our team to create project proposals that are more spot on. Points is open source and an ongoing project. We welcome contributions, and hope that our project can be useful to you!
```
[the remark this answers is not in the document]
Comment (FionaDL) [reply _posts/2020-10-01-introducing-points.markdown]: @lubc weird, I didn't think I touched that part of the sentence...

File: _posts/2020-10-01-introducing-points.markdown [hunk 4]
```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: "Introducing Points - An blind estimation tool from OmbuLabs"
```
Comment (lubc) [inline _posts/2020-10-01-introducing-points.markdown:3]:
```suggestion
title: "Introducing Points - A blind estimation tool from OmbuLabs"
```

**Review by lubc — CHANGES_REQUESTED** [review]
@FionaDL just added two more comments. Thanks!

**Review by lubc — APPROVED** [review]

**Pull request merged**